src/ui_ng/lib/src/service/interface.ts

```diff
@@ -66,6 +66,22 @@ export interface Tag extends Base {
 }
 
 /**
+ * Inteface for the tag view
```

Comment:
This mostly like a component data model, suggest to move it to the component declaration file.